workflow/models.py

```diff
@@ -149,10 +175,10 @@ class Parameter(models.Model):
         blank=True
     )
 
-    case = models.ForeignKey(
+    cases = models.ManyToManyField(
         'Case',
         related_name='parameters',
-        null=True,
+        through="CaseParameter",
```

Comment:
I had no clue you could do this :O

Reply:
Yep. can make for a lot of fun when your framework doesn't (seem?) to support it

Reply:
it kinda freaked when I did it, which is why the changes to the serializer methods.

Reply:
gotcha gotcha